### DISCUSSION TITLE
Randomise the position of windows opening to prevent them being all stacked into one

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#40] [Feature requests] [answered]
Currently when multiple feeds are opened at once MultiViewer will open them all in the middle of the screen.
![image](https://user-images.githubusercontent.com/25685344/173805039-f43f8bbd-90a9-48dd-9a3a-6fc1f88f437c.png)

This can make it difficult to identify which window is what and is a bit annoying.

**Comment by ManCaveMedia** [comment]
Nice suggestion.
Might be even better to default them to 1/3rd in size and start filling an imaginary 3x3 grid from the top left with each new window.

**Comment by tmdarlan** [comment]
I kinda like the in the middle but cascading offsets down and right. That way you can position the windows and its obvious which are placed and not placed

**Comment by f1multiviewer** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/106821994/174341738-01ce85a0-bb26-4821-b7e3-2f5972a3948a.png)

Basically this but with players? Coming right up!

Just kidding of course, but this staggered layout would work pretty well I think

**Reply by f1multiviewer** [reply]
Fixed as of v1.0.0-beta.10!

**Comment by tmdarlan** [comment]
Exactly.

Taylor Darland
